Eastern-Research-Group/expert-query
# Feature/340 update dependencies

## Related Issues:
* [EQ-340](https://jira.epa.gov/browse/EQ-340)

## Main Changes:
* Updated 3rd party dependencies.
* Updated one epa template, mainly to fix a console error from a script in the older version.

## Steps To Test:
1. Run the etl pointed at your local database
2. Verify the etl completes successfully
3. Start the app
4. Test as many features of the app

## TODO:
- [ ] Fix cypress tests
- [ ] Generate code coverage
